ormconfig.json

```diff
@@ -3,7 +3,7 @@
    "host": "localhost",
    "port": 5432,
    "username": "postgres",
-   "password": "mysecretpassword",
+   "password": "postgres",
```

Comment:
I think this may be a local value you've checked in

Reply:
Good spot! will revert this.